test_util/main.tf

```diff
@@ -76,6 +93,8 @@ module "dcos" {
   tags {
     owner = "${var.owner}"
     expiration = "${var.expiration}"
+    build_id = "${var.build_id}"
```

Comment:
@orsenthil does this works as expected and can you find the corresponding cost details in cost explorer?

Reply:
@DominikDary  - I don't how it correlates with cost details. But I will find out.

Reply:
And this conversation on measurement only indirectly related to the motivation here, which is about reducing the size of the instance. So, +1 with this change if the reduced instance size meets our testing criteria.